<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DHHCLPF18V2V2" failures="2" tests="15" name="Testcases.Testcommandbutton" time="144.190" errors="0" timestamp="2022-05-27T22:47:50 IST" skipped="0">
  <testcase classname="Testcases.Testcommandbutton" name="VerifysetandgetTooltip" time="9.509"/>
  <system-out/>
  <testcase classname="Testcases.Testcommandbutton" name="VerifyshowTooltip" time="19.841">
    <failure message="The following asserts failed:
	expected [true] but found [false],
	expected [true] but found [false],
	expected [true] but found [false]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false],
expected [true] but found [false],
expected [true] but found [false]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at PageActions.icommandactions.validateshowTooltip(icommandactions.java:62)
at Testcases.Testcommandbutton.VerifyshowTooltip(Testcommandbutton.java:39)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- VerifyshowTooltip -->
  <system-out/>
  <testcase classname="Testcases.Testcommandbutton" name="VerifyisVisible" time="11.181"/>
  <system-out/>
  <testcase classname="Testcases.Testcommandbutton" name="VerifyisEnable" time="7.417"/>
  <system-out/>
  <testcase classname="Testcases.Testcommandbutton" name="VerifyisToBedisplayed" time="7.568"/>
  <system-out/>
  <testcase classname="Testcases.Testcommandbutton" name="VerifygetViewstate" time="7.892"/>
  <system-out/>
  <testcase classname="Testcases.Testcommandbutton" name="VerifySetFocus" time="12.690"/>
  <system-out/>
  <testcase classname="Testcases.Testcommandbutton" name="VerifyFocus" time="7.706"/>
  <system-out/>
  <testcase classname="Testcases.Testcommandbutton" name="Verifyclk" time="7.381"/>
  <system-out/>
  <testcase classname="Testcases.Testcommandbutton" name="VerifySetVisible" time="8.179"/>
  <system-out/>
  <testcase classname="Testcases.Testcommandbutton" name="VerifySetDisplay" time="7.727"/>
  <system-out/>
  <testcase classname="Testcases.Testcommandbutton" name="VerifySetEnabled" time="7.736"/>
  <system-out/>
  <testcase classname="Testcases.Testcommandbutton" name="VerifySetandgetText" time="8.179"/>
  <system-out/>
  <testcase classname="Testcases.Testcommandbutton" name="VerifyChangeImage" time="13.433"/>
  <system-out/>
  <testcase classname="Testcases.Testcommandbutton" name="VerifyRemoveImage" time="7.751">
    <failure message="The following asserts failed:
	expected [false] but found [true]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [false] but found [true]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at PageActions.icommandactions.validateRemoveImage(icommandactions.java:194)
at Testcases.Testcommandbutton.VerifyRemoveImage(Testcommandbutton.java:110)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- VerifyRemoveImage -->
  <system-out/>
</testsuite> <!-- Testcases.Testcommandbutton -->
